checkstyle/suppressions.xml

```diff
@@ -18,7 +18,7 @@
     <suppress files="(ExpressionTreeRewriter|AstBuilder|KsqlParser|SchemaKTable).java"
               checks="ClassDataAbstractionCouplingCheck"/>
     <suppress
-            files="(QueryEngine|LogicalPlanner|FunctionRegistry|KsqlEngine|SchemaKStream|Analyzer|AggregateNode).java" checks="ClassDataAbstractionCouplingCheck"/>
+            files="(QueryEngine|LogicalPlanner|FunctionRegistry|KsqlEngine|SchemaKStream|Analyzer|AggregateNode|KsqlEngineMetrics).java" checks="ClassDataAbstractionCouplingCheck"/>
```

Comment:
I think its hard to justify disabling this check on KsqlEngineMetrics. It's not some class wiring together a load of others. Much better to look at how you can refactor the class. Otherwise, what's the point in the checkstyle rule if we're just going to disable it each time we hit it?

For example, if you change the class to use a lambda for the gauge, as I've suggested, you no longer need this suppression.

Reply:
Thanks for the suggestion Andy! Using lambda, there is no need for this suppression. Removed it!